src/components/Form/CreateEventForm.jsx

```diff
@@ -101,14 +102,9 @@ function CreateEventForm() {
     setLoading(false);
   };
 
-  const uploadJson = async () => {
-    //error handling
-    if (url.trim() == "" || name.trim() == "" || description.trim() == "") {
-      NotificationManager.warning(
-        "❗Please make sure all fields are completed before creating the event."
-      );
-      return;
-    } //make metadata
+  const uploadJson = async (url) => {
+ 
+   //make metadata
```

Comment:
[Boulder] Is there a reason you removed the validation from here

Reply:
yes, that's why I will add a form validation in the "onSubmit function"